aws-cpp-sdk-core/include/aws/core/client/AWSClient.h

```diff
@@ -153,6 +154,16 @@ namespace Aws
              */
             Aws::String GeneratePresignedUrl(Aws::Http::URI& uri, Aws::Http::HttpMethod method, const char* region, const char* serviceName, const char* signerName, const Aws::Http::HeaderValueCollection& customizedHeaders, long long expirationInSeconds = 0);
 
+#if AWS_SDK_VERSION_MAJOR > 1 || AWS_SDK_VERSION_MINOR >= 10
```

Comment:
dont we want this to be `&&`? technically [`0.14.9`](https://github.com/aws/aws-sdk-cpp/blob/0.14.9/aws-cpp-sdk-core/include/aws/core/VersionConfig.h#L16) would pass this check

Reply:
thanks

Reply:
I would just replace for exact match on 1.10, we only want to have it to work around merging this code before the last 1.9 release. After first release of 1.10 is done done, we should just remove the conditional behavior on version number.